nvflare/app_opt/pt/file_model_persistor.py

```diff
@@ -43,6 +43,7 @@ def __init__(
         source_ckpt_file_full_name=None,
         filter_id: str = None,
         load_weights_only: bool = False,
+        allow_numpy_conversion: bool = True,
```

Comment:
need to add a docstring. Why not specify a return format? E.g., `return_format = "pytorch"`

Reply:
what about `persist_format`?